pom.xml

```diff
@@ -88,6 +88,18 @@
       <type>jar</type>
       <scope>compile</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>mask-passwords</artifactId>
+      <version>2.8</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>junit</artifactId>
+      <version>1.10</version>
+    </dependency>
```

Comment:
`hudson.tasks.test` was moved into the junit plugin somewhere between version `1.573` and `1.599`